tests/test_csv_splitter.py

```diff
@@ -0,0 +1,72 @@
+from main import validate_file_length, is_input_valid, get_file_chunks, create_files
```

Comment:
I want you to think about whether these are unit tests or integration tests, and place them in a corresponding folder.